src/Processor/Gotenberg.php

```diff
@@ -41,7 +41,7 @@ protected function buildPdf(PrintAbstract $document, object $config): string
         $html = $document->renderDocument($params);
 
         $params['hostUrl'] = 'http://nginx:80';
-        if (isset($web2printConfig['gotenbergHostUrl'])) {
+        if (isset($web2printConfig['gotenbergHostUrl']) && !empty($web2printConfig['gotenbergHostUrl'])) {
```

Comment:
```suggestion
        if (!empty($web2printConfig['gotenbergHostUrl'])) {
```
empty() also cover isset()